releases.md

```diff
@@ -0,0 +1,38 @@
+# Proposed Revised Repo Model and Release Plan
+
+The Node.js TSC has recently been discussing and hammering out the details
+for a revised general release plan. The fundamental details of the plan are:
+
+<img src="https://camo.githubusercontent.com/675f052419c56a583c19644d4da30cc0ff4e02bb/68747470733a2f2f636c6475702e636f6d2f6c4d48746245334e72782d3330303078333030302e706e67"/>
+
+The `master` branch remains as the target of primary development. The majority of commits generally land there. The overall goal is to keep master as stable as possible, with periodic, numbered *alpha* releases being cut at fairly regular intervals.
```

Comment:
Should probably clarify stable vs API-somewhat-unstable / potentially breaking (but conservatively, we don't want to mess up the ecosystem hugely here either).

Also let's call it something more real and usable sounding than "alpha"? Maybe "future" or "next"?